R/tags.R

```diff
@@ -857,10 +865,13 @@ withTags <- function(code) {
 # Make sure any objects in the tree that can be converted to tags, have been
 tagify <- function(x) {
   rewriteTags(x, function(uiObj) {
-    if (isTag(uiObj) || isTagList(uiObj) || is.character(uiObj))
+    if (isTag(uiObj) || isTagList(uiObj) || is.character(uiObj)) {
       return(uiObj)
-    else
-      return(tagify(as.tags(uiObj)))
+    }
+    if (is_html_dependency(uiObj)) {
+      return(as.tags(uiObj))
+    }
```

Comment:
Needed to prevent infinite recursion now that as.tags.html_dependency() returns a tagList() of length 1